udata_front/theme/gouvfr/assets/js/pages/PublishingForm/PublishingForm.vue

```diff
@@ -103,10 +103,12 @@ const dataset = ref<NewDataset>({
     update_frequency: false,
     update_fulfilled_in_time: false,
   },
-  owner: null,
+  owner: user as User,
```

Comment:
Can you try to add a call to `auth()` from `js/plugins/auth.ts` to see if it remove the need to cast as User ?